nastafile.lisp

```diff
@@ -482,3 +482,11 @@ As the second value it returns ids of root articles.
   (dump-trees *all-articles*)
   (add-nojekyll-file))
 
+
+(defmain:defmain main
```

Comment:
Export the `main` function from the package.

Reply:
Cool. Is the usage correct there? I'll export the function and resubmit the PR.

Reply:
Just push new commit